tournaments/index.js

```diff
@@ -794,6 +794,7 @@ var commands = {
 			if (params.length < 1) {
 				return this.sendReply("Usage: " + cmd + " <minutes|off>");
 			}
+			if (params[0] === 'Infinity' || params[0] === '0') params[0] = 'off';
```

Comment:
Why? I wanted this to change Infinity to off in the modlog since /tour autodq off and /tour autodq Infinity have two different modlog messages, but do the same thing.